apps/mobile/src/utils/special-char.ts

```diff
@@ -0,0 +1,3 @@
+// A minus sign is a wider character than a hyphen, and should be used in
```

Comment:
Thanks for fixing this for me! I remember you flagged it previously but I forgot to do it

Reply:
Once you see you cannot unsee haha